src/dsp_tools/utils/logger_config.py

```diff
@@ -0,0 +1,45 @@
+import sys
+from datetime import datetime
+from pathlib import Path
+
+from loguru import logger
+
+logger_savepath = (Path.home() / ".dsp-tools" / "logger.log").absolute()
+
+
+def logger_config() -> None:
+    """
+    This function configures the log files.
+    Currently, there are three sinks:
+    - logger.log in the home directory for development purposes
+    - warnings.log in the cwd only with level warning and higher for the user (no stack-trace)
+    - print output on the terminal, formatted the same as the warnings.log
+    """
+    # If this is not removed, the default formatting is also printed out on the terminal
+    logger.remove()
+
+    text_format = "<level>{time:YYYY-MM:DD HH:mm:ss.SSS} | {level: <8} | {message}</level>"
```

Comment:
Sorry that I bring this up a second time, but I didn't see a reaction on my first comment. Are you sure you want the format `YYYY-MM:DD`? For me, `YYYY-MM-DD` would be more intuitive.

Reply:
Hey, I did not notice that comment then. Thanks for spotting this, this is wrong.